survey/SHACL_SurveyServiceOffering.ttl

```diff
@@ -0,0 +1,67 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix surveyonto: <http://semanticweb.org/metadatasurveyontology/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+surveyonto:SurveyServiceOfferingShape
+	a sh:NodeShape ; 
+	sh:targetClass surveyonto:SurveyServiceOffering ; 
+	sh:property [
+		sh:path surveyonto:survey_id ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example ID'" ;
+		sh:minCount 1 ; 
+		sh:message "Missing survey ID." ;
+		sh:name "Survey ID" ;
+	] ; 
+	sh:property [
+		sh:path surveyonto:survey_title ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example Title of Survey'" ;
+		sh:minCount 1 ;
+		sh:message "Missing Survey Title." ;
+		sh:name "Survey title" ;
+	] ; 
+	sh:property [
+		sh:path surveyonto:survey_description ; 
```

Comment:
you can reuse the description field as described in 
https://github.com/GAIA-X4PLC-AAD/ontology-management-base/issues/28
Maybe also applies for title (=name)?

Reply:
@robertschubert  please recheck if I did it correctly :)

Reply:
@phi981 
You do not need to duplicate the GeneralShape with the description into your SHACL. 
You can nest it into your SHACL. There is a section in the README how to nest the general Shape with `sh:node`.